packages/output-handler/src/index.ts

```diff
@@ -142,6 +94,7 @@ const processMessage = async (event: unknown) => {
 };
 
 const handler: Handler = async (event) => {
+	console.log(event);
```

Comment:
logger.info?

Reply:
I missed a few console. issues - resolved in https://github.com/guardian/transcription-service/pull/48/commits/db192aa2bc2486fec2a807641056aa17f5f33f2c (removed this particularly log line as I just needed it to get an updated message for `testMessage.ts` which is used to run the output-handler locally